paymentsheet/src/main/java/com/stripe/android/link/injection/LinkAccountManagerModule.kt

```diff
@@ -0,0 +1,42 @@
+package com.stripe.android.link.injection
+
+import androidx.annotation.VisibleForTesting
+import com.stripe.android.link.LinkConfiguration
+import com.stripe.android.link.account.DefaultLinkAccountManager
+import com.stripe.android.link.account.LinkAccountManager
+import com.stripe.android.link.analytics.LinkEventsReporter
+import com.stripe.android.link.repositories.LinkRepository
+import com.stripe.android.payments.core.analytics.ErrorReporter
+import dagger.Module
+import dagger.Provides
+
+@Module
+internal object LinkAccountManagerModule {
+
+    @Volatile
+    private var linkAccountManager: LinkAccountManager? = null
```

Comment:
This is not the right approach. If you want to synchronize between 2 activities, you should pass the data back and forth, and hold it until it's no longer needed.